client/my-sites/migrate/section-migrate.jsx

```diff
@@ -62,9 +63,11 @@ class SectionMigrate extends Component {
 	resetMigration = () => {
 		const { targetSiteId, targetSiteSlug } = this.props;
 
+		this.setMigrationState( { migrationStatus: 'unknown', errorMessage: '' } );
+		this.updateFromAPI();
```

Comment:
I've moved this call from the reset `then` callback to here so the normal view returns as quickly as possible on reset: if we wait for the response from the `resetMigration` call there's a slight delay.